lib/top_author.rb

```diff
@@ -0,0 +1,16 @@
+class Top_Author < Author
+		attr_accessor :age, :name
+	def initialize(args)
+			super
+			@best_book = args[:best_book]
+	end
+
+	def welcome
+		puts "Hello to our famous author, #{@name}."
+	end
+
+	def top_seller
+		puts "The top seller is #{@best_book}."
+		
+	end
+end
```

Comment:
I was having some issues with getting my classes set so I created a new one
for testing and to make sure that I could do it; not really doing anything
necessary

On Tue, Jan 27, 2015 at 11:52 AM, Gerry Pass <notifications@github.com>
wrote:

> In lib/top_author.rb
> <https://github.com/laingbeth/talkative_robot/pull/7#discussion_r23622541>
> :
>
> > +class Top_Author < Author
> > +		attr_accessor :age, :name
> > +	def initialize(args)
> > +			super
> > +			@best_book = args[:best_book]
> > +	end
> > +
> > +	def welcome
> > +		puts "Hello to our famous author, #{@name}."
> > +	end
> > +
> > +	def top_seller
> > +		puts "The top seller is #{@best_book}."
> > +		
> > +	end
> > +end
>
> Not sure what this does.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/laingbeth/talkative_robot/pull/7/files#r23622541>.
>



-- 
Beth Laing
<http://www.linkedin.com/in/bethlaing>